### PR TITLE
[lldp] Clean up service start logic owing to port init start optimization

### PR DESCRIPTION
**Pull request by vivekrnv**
#### Why I did it

As part of service start optimization done, we've removed timer units and used PORT_INIT_DONE as the checkpoint for starting delayed services. Thus this logic is redundant

This fixes the following error log. 
```
Nov  3 14:16:54.281546 r-leopard-44 ERR systemctl[10553]: Failed to get unit file state for lldp.timer: No such file or directory
```

#### How I did it

Redundant lldp timer code is removed from syncd.sh. 

#### How to verify it

Build and test 

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Review by dgsudharsan — APPROVED** [review]

File: files/build_templates/lldp.timer.j2 [hunk 1]
```diff
@@ -1 +0,0 @@
-per_namespace/lldp.timer.j2
```
Comment (saiarcot895) [inline files/build_templates/lldp.timer.j2:1]: Does the actual file need to be delelted as well, not just the symlink?

Comment (vivekrnv) [reply files/build_templates/lldp.timer.j2:1]: Actual file doesn't exist

**Review by saiarcot895 — APPROVED** [review]

File: files/build_templates/lldp.timer.j2 (hunk 1 above)
Comment (yxieca) [reply files/build_templates/lldp.timer.j2:1]: The file is here: https://github.com/sonic-net/sonic-buildimage/blob/master/files/build_templates/per_namespace/lldp.service.j2

Removing the symbolic link might break chassis? @arlakshm

**Review by yxieca — CHANGES_REQUESTED** [review]
@arlakshm to assess the removal of lldp.timer.j2

File: files/build_templates/lldp.timer.j2 (hunk 1 above)
Comment (saiarcot895) [reply files/build_templates/lldp.timer.j2:1]: That's the `lldp.service` file, not the `lldp.timer` file. It looks like the timer was removed in #13969.

**Comment by arlakshm** [comment]
> @arlakshm to assess the removal of lldp.timer.j2

@SuvarnaMeenakshi @abdosi can help check if this change will have impact on chassis bootup?

**Comment by vivekrnv** [comment]
@SuvarnaMeenakshi @abdosi  @yxieca Any update here?

**Comment by SuvarnaMeenakshi** [comment]
I don't see this change impacting multi-asic or chassis apart from the fact that lldp will always start after the delay which was not the case earlier (not due to this PR but due to PR 13969)
This PR is more to clean up the change that was missed in the enhancement PR.
The script change here is checking if lldp.timer is enabled, if it is enabled then it would let the timer wait and start service upon warm-boot or fast-boot it will start immediately if for cold boot. 
With the enhancement changes in PR 13969, lldp is a delayed service and will start after the delay for all scenarios and all platforms. 
Is that acceptable? @yxieca

**Pull request merged**